### DISCUSSION TITLE
Editing a connection fails to set up a new driver and configuration

### DISCUSSION THREAD

**Discussion by zghlx** [dbeaver/dbeaver#21742] [Q&A]
### Description

![image](https://github.com/dbeaver/dbeaver/assets/5928564/3b120083-c9aa-4e84-8986-c7467d32ee0b)

Assuming that the driver information of the connection is lost due to copying some files in the workspace, how can I specify the driver and settings of the new configuration by editing the connection?

### DBeaver Version

23.2.3

**Comment by zghlx** [comment]
I have too many exists connections were lost original driver settings.   Is there any method to batch change the driver settings ？

**Comment by zghlx** [comment]
> I have too many exists connections were lost original driver settings. Is there any method to batch change the driver settings ？

or temp method?

**Comment by LonwoLonwo** [comment]
Hello @zghlx 

Did you try the Database -> Change Connection(s) Driver option?

**Comment by zghlx** [comment]
> Hello @zghlx
> 
> Did you try the Database -> Change Connection(s) Driver option?
I finally found it, thank you.
